File: Android/app/src/main/java/com/nfsindustries/cryptocurrencymonitor/cryptocurrencymonitor/TextFormatter.java
package com.nfsindustries.cryptocurrencymonitor.cryptocurrencymonitor;

/**
 * Created by maxmo on 26/5/2017.
 */

public final class TextFormatter {

    /**
     * Format currency to display with only 2 decimal places
     * @param value currency value
     * @return formatted String to display
     */
    public static final String formatCurrency(final float value) {
        return String.format("%.2f", value);
    }
}

File: Android-Kotlin/app/src/main/java/com/nfsindustries/cryptocurrencymonitor/utils/TextFormatter.kt
package com.nfsindustries.cryptocurrencymonitor.utils

/**
 * Static class used to format text, strings and currencies to display
 */

object TextFormatter {

    /**
     * Format currency to display with only 2 decimal places
     * @param value currency value
     * @return formatted String to display
     */
    fun formatCurrency(value: Float): String {
        return String.format("%.2f", value)
    }
}
